src/components/autocomplete/autocomplete.scss

```diff
@@ -110,9 +110,17 @@ md-autocomplete {
     padding: 0 15px;
     line-height: 40px;
     height: 40px;
+    position: absolute;
+  }
+  .md-autocomplete-placeholder {
+    color: #757575;
+    font-size: 14px;
+    line-height: 40px; // Matches md-autocomplete height.
+    padding: 0 15px;
   }
   .md-show-clear-button button {
-    position: relative;
+    position: absolute;
+    right: 0;
```

Comment:
Ah; will edit the following into the description:

CSS positioning is used to ensure that the layering order is as follows (from bottom to top):
autocomplete container
placeholder
text input field
This way, the placeholder text is on top of the white background, but behind the transparent text input, allowing us to see it, yet still be able to click on the text field.